config/routes.yml

```diff
@@ -0,0 +1,42 @@
+admin_link_widget_list:
+  path: /link_widget/list
+  methods: [GET]
+  defaults:
+    _controller: 'PrestaShop\LinkList\Controller\Admin\Improve\Design\LinkWidgetController::listAction'
+    _legacy_controller: AdminLinkWidget
+#    _legacy_link: AdminLinkWidget
```

Comment:
it is one of the two solutions I experimented to update the `Link::createUrl` method, we discussed about it with @mickaelandrieu and it will be used I will link the PR from the core when I have cleaned it